CONTRIBUTING.md

```diff
@@ -9,3 +13,65 @@
 - [Visual Studio Code](https://code.visualstudio.com/)
 - [Microsoft Graph PowerShell SDK](https://github.com/microsoftgraph/msgraph-sdk-powershell), cloned into the same parent folder of this repository. This dependency is only required if you want to generate SDKs for Microsoft Graph.
 - [reportgenerator](https://www.nuget.org/packages/dotnet-reportgenerator-globaltool), if you want to be able to generate coverage reports from the pre-configured visual studio code test tasks.
+
+## Building the project
+
+```sh
+dotnet restore ./src/kiota
+dotnet build ./src/kiota
+```
+
+## Running the tests
+
+### Test the kiota command line
+
+
+```sh
+dotnet test ./tests/Kiota.Tests/
```

Comment:
```suggestion
dotnet test
```
Do you see any advantage of running the tests separately instead of all at once?

Reply:
Yes -- it allows me to avoid installing wasm tools.
```
>dotnet test
  Determining projects to restore...
/usr/local/share/dotnet/sdk/7.0.304/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.Sdk.ImportWorkloads.targets(38,5): error NETSDK1147: To build this project, the following workloads must be installed: wasm-tools [/Users/mikekistler/Projects/Microsoft/kiota/src/Kiota.Web/Kiota.Web.csproj]
/usr/local/share/dotnet/sdk/7.0.304/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.Sdk.ImportWorkloads.targets(38,5): error NETSDK1147: To install these workloads, run the following command: dotnet workload restore [/Users/mikekistler/Projects/Microsoft/kiota/src/Kiota.Web/Kiota.Web.csproj]
```